### PR TITLE
Privacy followup

### PR DESCRIPTION
**Pull request by ineilson**
Addresses the 3 primary (bulleted) issues in #366.

### CODE REVIEW

File: lib/Gocdb_Services/User.php [hunk 1]
```diff
@@ -118,18 +118,53 @@ public function isAllowReadPD(\User $user) {
...
 
-        foreach ($sites as $site) {
-            if ($authServ->authoriseAction(\Action::READ_PERSONAL_DATA, $site, $user)
+        foreach ($entities as $entity) {
+            if ($authServ->authoriseAction(\Action::READ_PERSONAL_DATA, $entity, $user)
                 ->getGrantAction()
             ) {
                 // exit the first time we find a grant as we don't support
-                // site-level viewing granularity.
+                // site or ngi-level viewing granularity.
```
Comment (gregcorbett) [inline lib/Gocdb_Services/User.php:163]:
```suggestion
                // site, ngi or project-level viewing granularity.
```

File: htdocs/web_portal/controllers/ngi/view_ngi.php [hunk 2]
```diff
@@ -36,15 +36,9 @@ function view_ngi() {
...
-    if(!is_null($user)) {
-        $params['UserIsAdmin']=$user->isAdmin();
-    }
-
-    $params['authenticated'] = false;
-    if($user != null){
-        $params['authenticated'] = true;
-    }
+    // Set values for showing personal data
+    $params = array();
```
Comment (gregcorbett) [inline htdocs/web_portal/controllers/ngi/view_ngi.php:40]: Does this "reset" `$params` to an empty array?

Comment (ineilson) [reply htdocs/web_portal/controllers/ngi/view_ngi.php]: Good spot, thanks.

**Comment by ineilson** [comment]
> Can you address the Codacy issues?

I could but can we remove (or reverse) _Opening brace should be on a new line_ from the ruleset (where is the codesniffer ruleset stored?)? It is not the current practice throughout gocdb codebase and IMHO enforcing it now has no real benefit.

**Comment by gregcorbett** [comment]
The current practice is bad and it should feel bad 😋. We should write new code that follows PSR-12 so that it's easier for all to understand.

The existing code will take time to bring up to standard, but we can get there 😄

**Comment by gregcorbett** [comment]
> Can you address the Codacy issues?

Addressed in #372, along with wider PSR-12 improvements to the files touched in this PR.

**Review by gregcorbett — APPROVED** [review]
LGTM

**Pull request merged**